tests/integration/ambient/baseline_test.go

```diff
@@ -2425,3 +2606,28 @@ func TestDirect(t *testing.T) {
 		})
 	})
 }
+
+func deleteWaypoints(t framework.TestContext, nsConfig namespace.Instance, sa string) {
```

Comment:
This can probably be removed now unless we have an idea of where the original author intended to use it (e.g. additional test case).

Reply:
I meant to remove that, the case isn't relevant now with recent changes